### PR TITLE
TestSuite PoC - Remove Test prefix from annotations

### LINKED ISSUE #31522: Testsuite PoC - Rename annotiations to drop `Test` prefix
Currently all annotations are prefixed with `Test`, e.g. `@TestWebDriver`, `@TestPage`, `@TestRealm`, etc. The `Test` prefix conveys no semantic meaning other than 'this is used in a test' and probably needs a better name. 

Since the annotation is used as an injection point for the supplier it  might be better to use `Inject` as it more accurately represents the fact this injects something into the test. For example `@InjectWebDriver`, `@InjectPage`, `@InjectRealm`, etc.

### PR DESCRIPTION
**Pull request by jonkoops**
Closes #31522

### CODE REVIEW

File: test-poc/examples/src/test/java/org/keycloak/test/examples/WelcomePageTest.java [hunk 1]
```diff
@@ -3,14 +3,13 @@
...
-import org.openqa.selenium.WebDriver;
+import org.keycloak.test.framework.annotations.WebDriver;
 
 @KeycloakIntegrationTest(config = NoAdminUserKeycloakTestServerConfig.class)
 public class WelcomePageTest {
 
-    @TestWebDriver
-    WebDriver driver;
+    @WebDriver
+    org.openqa.selenium.WebDriver driver;
```
Comment (vaceksimon) [inline test-poc/examples/src/test/java/org/keycloak/test/examples/WelcomePageTest.java:12]: So we have a little conflict. But I think this is fine

**Review by vaceksimon — APPROVED** [review]
I'm not a fan of what it did to references to the selenium `WebDriver` class, but I can live with it

File: test-poc/examples/src/test/java/org/keycloak/test/examples/WelcomePageTest.java (hunk 1 above)
Comment (miquelsi) [reply test-poc/examples/src/test/java/org/keycloak/test/examples/WelcomePageTest.java:12]: Can we think of a better name for this case? I'm also not convinced with the conflict

Comment (vaceksimon) [reply test-poc/examples/src/test/java/org/keycloak/test/examples/WelcomePageTest.java:12]: Maybe `WebBrowser`, `Browser`, `BrowserDriver`, or just `Driver`? :car:

**Review by stianst — CHANGES_REQUESTED** [review]
What happened to WebDriver makes me re-consider this naming convention, and I propose we continue to have a prefix. That also would help to differ these type of annotations from other annotations.

I propose we stick with `@TestRealm`, or change to `@InjectRealm`.

**Comment by jonkoops** [comment]
> I propose we stick with `@TestRealm`, or change to `@InjectRealm`.

Created #31570 to rename this to `Inject`

**Comment by lhanusov** [comment]
Can you please close this one? @jonkoops.

File: test-poc/examples/src/test/java/org/keycloak/test/examples/WelcomePageTest.java (hunk 1 above)
Comment (Aboullos) [reply test-poc/examples/src/test/java/org/keycloak/test/examples/WelcomePageTest.java:12]: What usually did was name it DriverHelper

**Comment by jonkoops** [comment]
I'll close it if we decide to merge the other one :)

**Review by stianst — CHANGES_REQUESTED** [review]
Let's go with the `Inject` prefix. @jonkoops can you update the PR?

**Comment by stianst** [comment]
#31570 was merged, so closing this one

**Pull request closed without merging**